dietpi/dietpi-services

```diff
@@ -194,6 +194,9 @@ Available services:
 			'mycroft'
 			'fahclient'
 			'cron'
+      
+      # - DNS
+      'unbound'
```

Comment:
It is dangerous to control the DNS server, as it might be used by the system itself and the we break name resolution. For that reason we do not control Pi-hole. But, it could be also made a special case: Control it as long as `nameserver 127.0.0.1` is not found in `/etc/resolv.conf`? Requires some more code change in `dietpi-services`, so I'd do that in a separate PR. For now I suggest you add it below `pihole-FTL` in the special block below that is only to show status or when explicitly calling `dietpi-services stop unbound`.